ui/src/overridden/frontsite/Document/DocumentDetails/DocumentItemBody.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { DocumentItemBody } from "@inveniosoftware/react-invenio-app-ils";
+import _get from "lodash/get";
+import { parametrize } from "react-overridable";
+import { Table } from "semantic-ui-react";
+
+export const DocumentItemTableHeader = () => {
+  return (
+    <Table.Row data-test="header">
+      <Table.HeaderCell>Barcode</Table.HeaderCell>
+      <Table.HeaderCell>Shelf</Table.HeaderCell>
+      <Table.HeaderCell>Call Number</Table.HeaderCell>
+      <Table.HeaderCell>Status</Table.HeaderCell>
+      <Table.HeaderCell>Medium</Table.HeaderCell>
+      <Table.HeaderCell>Loan restriction</Table.HeaderCell>
+    </Table.Row>
+  );
+};
+
+function renderShelflink(item) {
+  const shelfNumber = _get(item, "shelf");
+  if (shelfNumber === undefined) return;
+  const shelfLink = `https://maps.web.cern.ch/?n=['SHELF ${shelfNumber}']`;
+  return (
+    <a href={shelfLink} target="_blank" rel="noreferrer">
```

Comment:
should we add a pin icon? and small text "See on the map"? Otherwise it will not be very prominent, and we should advertise new cool features :)